packages/nodejs/package.json

```diff
@@ -26,17 +25,26 @@
     "redis": "*"
   },
   "scripts": {
-    "prebuild": "npm run versionfile",
     "build": "tsc -p tsconfig.json",
-    "build:watch": "npm run versionfile && tsc -p tsconfig.json -w --preserveWatchOutput",
-    "clean": "rimraf dist coverage",
+    "build:ext": "node-gyp rebuild",
```

Comment:
```suggestion
```
I would just remove this. It's like only calling "make" for the Ruby extension installation. That doesn't work if you haven't done all the other steps beforehand already.